src/query/db_accessor.hpp

```diff
@@ -459,6 +459,8 @@ class DbAccessor final {
   }
 
   storage::Result<std::optional<VertexAccessor>> RemoveVertex(VertexAccessor *vertex_accessor) {
+    accessor_->PrefetchOutEdges(vertex_accessor->impl_);
```

Comment:
The test which failed here now uses `DetachDelete` function.
But we still call this function in `mgp_graph_delete_vertex` without prefetching edges.
Also `DetachDelete` prefetches edges and call `accessor_->DetachDelete()` internally.
Here `accessor_->DeleteVertex()` calls `accessor_->DetachDelete` also.
So we must prefetch edges.